pkgs/development/python-modules/yamlpath/default.nix

```diff
@@ -0,0 +1,46 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, fetchPypi
+, isPy3k
+, ruamel-yaml
+}:
+let
+  pinned-ruamel-yaml = ruamel-yaml.overridePythonAttrs (
```

Comment:
This is not allowed for packages that are under pythonPackages.

Reply:
It doesn’t work with the version in nixpkgs (which is newer). What do you suggest to do instead?
`overridePythonAttrs` is already used for 6 packages inside python-modules.

Reply:
Thanks for the review, I moved the package out of python-modules.